run.sh

```diff
@@ -346,6 +349,23 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  if [[ ! "${REQUIRE-}" ]]; then
+    h3 "No package dependencies listed"
+    STATUS SKIP
+    return
+  fi
+
+  h3 "Installing composer packages"
+  touch /app/composer.json
```

Comment:
Please see my second comment above: detecting REQUIRE is far easier than finding composer.json